CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 * chore(deps) [\#1141](https://github.com/Finschia/finschia-sdk/pull/1141) Bump github.com/cosmos/ledger-cosmos-go from 0.12.2 to 0.13.2 to fix ledger signing issue
+* (consensus) [\#1179](https://github.com/Finschia/finschia-sdk/pull/1179) modify missing changes of converting to tendermint
```

Comment:
It would be better to use x/slashing not consensus, because it will not affect consensus itself.

Reply:
In this PR, the changed codes are `x/auth` and `x/slashing` modules. 
If then, I think we need to add all module in the category. How do you thinking about this?

Reply:
Listing both of them would also be a good choice.